### PR TITLE
chore(web-server): use a rewriting server to get rid of bower dependency

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,32 +6,39 @@
...
     "protractor": "protractor e2e-tests/protractor.conf.js",
-
-    "update-index-async": "node -e \"var fs=require('fs'),indexFile='app/index-async.html',loaderFile='app/bower_components/angular-loader/angular-loader.min.js',loaderText=fs.readFileSync(loaderFile,'utf-8').split(/sourceMappingURL=angular-loader.min.js.map/).join('sourceMappingURL=bower_components/angular-loader/angular-loader.min.js.map'),indexText=fs.readFileSync(indexFile,'utf-8').split(/\\/\\/@@NG_LOADER_START@@[\\s\\S]*\\/\\/@@NG_LOADER_END@@/).join('//@@NG_LOADER_START@@\\n'+loaderText+'    //@@NG_LOADER_END@@');fs.writeFileSync(indexFile,indexText);\""
+    "update-index-async": "node -e \"var fs=require('fs'),indexFile='app/index-async.html',loaderFile='app/lib/angular-loader/angular-loader.min.js',loaderText=fs.readFileSync(loaderFile,'utf-8').split(/sourceMappingURL=angular-loader.min.js.map/).join('sourceMappingURL=lib/angular-loader/angular-loader.min.js.map'),indexText=fs.readFileSync(indexFile,'utf-8').split(/\\/\\/@@NG_LOADER_START@@[\\s\\S]*\\/\\/@@NG_LOADER_END@@/).join('//@@NG_LOADER_START@@\\n'+loaderText+'    //@@NG_LOADER_END@@');fs.writeFileSync(indexFile,indexText);\""
```
Comment (gkalpak) [inline package.json]: `loaderFile='app/lib/angular-loader/angular-loader.min.js` should point to `node_modules/` instead.

File: app/index.html [hunk 2]
```diff
@@ -9,10 +9,10 @@
   <title>My AngularJS App</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="bower_components/html5-boilerplate/dist/css/normalize.css">
-  <link rel="stylesheet" href="bower_components/html5-boilerplate/dist/css/main.css">
+  <link rel="stylesheet" href="lib/html5-boilerplate/dist/css/normalize.css">
```
Comment (mgol) [inline app/index.html]: Why os this needed? Previously the files were served from `bower_components`, why can't they just now be served from `node_modules` without any rewriting server?

Comment (gkalpak) [inline app/index.html]: Because the root directory (for the webapp) is `app/` which is a sibling of `node_module/`.

Comment (mgol) [inline app/index.html]: I see. And in `package.json` it's not possible to specify an alternative location of `node_modules`.

Comment (mgol) [inline app/index.html]: We could create a separate package.json inside app. ;)

Comment (petebacondarwin) [inline app/index.html]: We could. I did think about that. But I felt that might be more confusing than helpful.
And in reality, and real project is going to have a build step of some kind, right?

Comment (gkalpak) [inline app/index.html]: I would rather not introduce a task runner (grunt, gulp etc). Everyone will have a different preference and this seed project has been tranditionally quite unopinionated. Also, keep n mind that `angular-seed` is used as a base for `angular-phonecat`, so simplicity is a feature :smiley:

How about using an npm script (e.g. postinstall) that copies the necessary files from `node_modules/...` to `app/lib/`? It is still less straightforward than `bower install` imo, but more straightforward than using a task runner or a rewriting server. WDYT?

Comment (mgol) [inline app/index.html]: @gkalpak That's exactly what I proposed in my last comment...

Comment (gkalpak) [inline app/index.html]: Ooops, I thought you were proposing to use Grunt :grin:

**Comment by gkalpak** [comment]
Here is the alternative approach discussed above: #389

File: app/index.html (hunk 2 above)
Comment (mgol) [inline app/index.html]: No, I just gave an example of how jQuery does it. :) The details of the copying mechanisms are not the most important parts.

**Comment by petebacondarwin** [comment]
Closing as there is no interest in landing this.

**Pull request closed without merging**